static/app/views/dashboardsV2/widgetCard/index.tsx

```diff
@@ -282,6 +289,14 @@ class WidgetCard extends Component<Props, State> {
                 </Tooltip>
                 {this.renderContextMenu()}
               </WidgetHeader>
+              {hasSessionDuration && (
+                <PanelAlert type="warning">
+                  {t(
+                    'session.duration is no longer being recorded as of %s. Data in this widget may be incomplete.',
+                    getFormattedDate(SESSION_DURATION_INGESTION_STOP_DATE, 'MMM D, YYYY')
+                  )}
+                </PanelAlert>
+              )}
```

Comment:
We could make this reusable so that we don't duplicate hasSessionDuration logic and the Panel code in `static/app/components/modals/widgetViewerModal.tsx` but considering this will be gone soon I don't think it's a big deal.